pom.xml

```diff
@@ -151,11 +151,6 @@
         <artifactId>proto-google-common-protos</artifactId>
         <version>2.57.0</version>
       </dependency>
```

Comment:
@enocom I'm guessing we might still need something like this since we still use [classes from the graalvm-sdk dependency](https://github.com/GoogleCloudPlatform/alloydb-java-connector/blob/1eae582332fe51aa01a02d9ceff4c6acc252ca97/alloydb-jdbc-connector/src/main/java/com/google/cloud/alloydb/nativeimage/AlloyDBFeature.java#L29)?
```
    <dependency>
      <groupId>org.graalvm.sdk</groupId>
      <artifactId>graal-sdk</artifactId>
      <version>${graal-sdk.version}</version>
      <scope>provided</scope>
    </dependency>
```

Example: https://github.com/googleapis/java-spanner/blob/10c35639b3a82fab5d19e7f02c7032c59e86d2df/google-cloud-spanner/pom.xml#L359C1-L364C18 in the spanner repo

Reply:
{graal-sdk.version} is a property defined in java-shared-config.